docroot/modules/humsci/hs_admin/hs_admin.module

```diff
@@ -191,14 +191,40 @@ function hs_admin_preprocess_page(&$variables) {
 /**
  * Implements hook_field_widget_complete_WIDGET_TYPE_form_alter().
  */
-function hs_admin_field_widget_complete_paragraphs_browser_form_alter(&$field_widget_complete_form, FormStateInterface $form_state, $context) {
+function hs_admin_field_widget_complete_paragraphs_form_alter(&$field_widget_complete_form, FormStateInterface $form_state, $context) {
   $max_delta = $field_widget_complete_form['widget']['#max_delta'] ?? -1;
   for ($delta = 0; $delta <= $max_delta; $delta++) {
     if (isset($field_widget_complete_form['widget'][$delta]['top']['actions']['dropdown_actions']['duplicate_button'])) {
       $duplicate_button = &$field_widget_complete_form['widget'][$delta]['top']['actions']['dropdown_actions']['duplicate_button'];
       $duplicate_button['#ajax']['callback'] = 'su_humsci_profile_paragraphs_duplicate_callback';
+      // Remove icon from paragraph items in node edit form.
+      unset($field_widget_complete_form['widget'][$delta]['top']['type']['icon']);
     }
   }
+  // Change the add paragraph button in flexible page, private page and
+  // collection fields to match the previously used paragraph browser viersion.
```

Comment:
```suggestion
  // Change the add paragraph button in flexible page, private page and
  // collection fields.
```
I wouldn't reference something that's being deprecated in a comment. Future coders may not understand the reference. I'd either drop it (like suggestion) or specify the actual behavior.

Reply:
Fixed.